crates/orderbook/src/database/orders.rs

```diff
@@ -318,6 +318,11 @@ fn calculate_status(order: &FullOrder) -> OrderStatus {
     if order.valid_to < Utc::now().timestamp() {
```

Comment:
Ubernit: Maybe we should:
```rust
let now = Utc::now().timestamp();
```

And then use `now` for both checks. Really tiny (and OK with merging as-is).